File: JavaScript/Iniciante/1007.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

a = parseInt(lines.shift())
b = parseInt(lines.shift())
c = parseInt(lines.shift())
d = parseInt(lines.shift())

diferenca = a * b - c * d

console.log(`DIFERENCA = ${diferenca}`)

File: R/Iniciante/1007.r
input <- file('stdin', 'r')

a <- as.integer(readLines(input, n = 1))
b <- as.integer(readLines(input, n = 1))
c <- as.integer(readLines(input, n = 1))
d <- as.integer(readLines(input, n = 1))

diferenca = (a * b - c * d)

cat(sprintf("DIFERENCA = %d\n", diferenca))
